### DISCUSSION TITLE
Aditional example code

### DISCUSSION THREAD

**Discussion by Bach-Danielsson** [eModbus/eModbus#84] [Ideas]
First thank you for a greate work making this eModbus.

Can someone add an example using esp32 as server with the funktion set coil on GPIO 32  and read coil GPIO 33.
also including the possibility to have the function READ_HOLD_REGISTER.

**Comment by Miq1** [comment]
I am afraid you will have to code that yourself - this is exactly what the library is intended for. I would suggest you first should try to understand how the existing examples are working, then adapt to other function codes that you will need. It is not that difficult at all! 

You can always share your code here to have us have a look at it of course.
